draft-ietf-quic-manageability.md

```diff
@@ -877,30 +877,6 @@ and ACK manipulation is impossible. Specifically, heuristics attempting to
 distinguish ACK-only packets from payload-carrying packets based on packet size
 are likely to fail, and are emphatically NOT RECOMMENDED.
 
-## QoS support and ECMP
-
-\[EDITOR'S NOTE: this is a bit speculative; keep?]
-
-QUIC does not provide any additional information on requirements on Quality of
-Service (QoS) provided from the network. QUIC assumes that all packets with the
-same 5-tuple {dest addr, source addr, protocol, dest port, source port} will
-receive similar network treatment.  That means all stream that are multiplexed
-over the same QUIC connection require the same network treatment and are handled
-by the same congestion controller. If differential network treatment is desired,
-multiple QUIC connections to the same server might be used, given that
-establishing a new connection using 0-RTT support is cheap and fast.
```

Comment:
I think we should keep this first part here. Especially the part saying "If differential network treatment is desired,
 multiple QUIC connections to the same server might be used" because the idea that QUIC packets of the same connection that carry different stream data could have different DCSPs came up more than once that that really is a bad idea for CC.

Reply:
That is probably a deep enough conversation that I don't think it is worth including here.  Multiple connections might not give you what you want either.  And different congestion controllers might tolerate variations in QoS better.  The problem with all of this text is that it is creating the impression that underlying semantics can be inferred.  As a general theme, that is not good and so saying anything more than what is strictly factual is likely to encourage ossification of various sorts.

Reply:
So what I would like to say somewhere if that if an endpoint want different network treatment based on DSCP (ignoring that DSCP is not usable e2e) it should use different connections and set the same DSCP on all packets of the same connection, instead of getting the idea of setting different DSCP on different packets of the same connection.

Reply:
Also I think the guidance for ECMP is still to schedule based on 5-tuple (and not on a per-packet base). I think that is still true for QUIC and we should state that.

As such if an endpoint wants to somehow make use of ECMP (e.g. in a datacenter setup) on need to consider to use different connections.

I think we can also say that it is generally recommended to use as few QUIC connections as possible to the same server (for CC reasons).